README.md

```diff
@@ -25,6 +25,21 @@ As of v0.5.x, this plugin now requires Prettier v3 and is ESM-only. This means i
 
 ## Options
 
+### Using with Tailwind CSS v4.0
+
+When using Tailwind CSS v4 you must specify what CSS file we should use that contains your theme, custom utilities, etc… To do this use the `tailwindStylesheet` option in your Prettier configuration.
```

Comment:
Suggested wording, mostly to avoid saying "we should use" — while I get it, that wording feels kind of weird 😅

```suggestion
When using Tailwind CSS v4 you must specify your CSS file entry point, which includes your theme, custom utilities, and other Tailwind configuration options. To do this, use the `tailwindStylesheet` option in your Prettier configuration.
```